cmfive.php

```diff
@@ -263,8 +262,7 @@ function refreshComposerAvailability()
 
 function cmdinstallCoreLibraries($pCount, $parameters = [])
 {
-    $parameters = array_slice($parameters, 2);
-    installCoreLibraries($parameters[0]);
+    installCoreLibraries(($pCount > 2) ? $parameters[2] : null);
```

Comment:
```$pCount``` could be specified completely unrelated to ```$parameters```, ideally we should be counting ```$parameters``` and not relying on the count specified just in case they don't match in certain scenarios.

Reply:
Generally yes, but this function is purely a stub/wrapper to interface between CLI and a generic function (installCoreLibraries).
So it should only ever be called under the CLI convention of argc, argv.
Otherwise installCoreLibraries should be called directly.
There is another of these in cmfive.php = cmdSeedAdminUser, vs cmfivetests & toolsdb which alleviate the need by having a "genericRunner" function to moderate menu vs CLI.